lookup_plugins/azure_keyvault_secret.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: azure_keyvault_secret
+    author:
+        - Hai Cao <t-haicao@microsoft.com>
+    version_added: 2.7
+    requirements:
+        - requests
+        - azure
+        - msrest
+    short_description: read secret from Azure Key Vault.
+    description:
+      - This lookup returns the content of a secret kept in Azure Key Vault.
+    options:
+        _terms:
+            description: secret name of the secret to retrieve, version can be included like secret_name/secret_version.
+            required: True
+        vault_url:
+            description: url of Azure Key Vault to be retrieved from
+            required: True
+        client_id:
+            description: client_id of service principal that has access to the provided Azure Key Vault
+        secret:
+            description: secret of service principal provided above
+        tenant_id:
+            description: tenant_id of service principal provided above
+    notes:
+        - If version is not provided, Key Vault will give the latest version.
+        - If ansible is running on Azure Virtual Machine with MSI enabled, client_id, secret and tenant isn't necessary.
+        - For how to enable MSI on Azure VM, please refer to this doc https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/
```

Comment:
"For how to enable" could be change to "For enabling", or even better "To enable"